### ISSUE TITLE
How old are you + cinema package

### ISSUE THREAD

**Issue by gnouf1** [leon-ai/leon#107]
### Feature Use Case
- How old are you ?
- What are new movies of this week ?

### Feature Proposal
One module "How old are you ? " and an cinema/serie package.
I have time so I can do both of them, can I ? :)

**Comment by louistiti** [comment]
Hello @gnouf1,

Thanks for your suggestions!

I'll soon work on the "How Old" module under the "Leon" package. But feel free to kick off the "TV" package. I've noted [some ideas](https://trello.com/c/JB5gdQrV/61-%E2%9C%A8-tv-package) on the roadmap.

**Comment by gnouf1** [comment]
Yes i see ideas on your trello ;p
In parallel with this module I will probably work on the trust website module :) (i see somebody open an issue). If it's ok for you :)

**Comment by louistiti** [comment]
Alright then!

Feel free to ask questions if you need to.

**Comment by ThomasRitaine** [comment]
Hello !

The PR for **How old are you ?** is : #142

**Comment by Divlo** [comment]
Thanks for your report! @gnouf1 
Could you please reopen another issue for the Cinema package, we should keep unrelated things separated, it's easier to track! :smile:

**Comment by Divlo** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [✨ [1.0.0-beta.7] How Old Module - Leon Package (JavaScript)](https://trello.com/c/8Xs4j3Mr/279-%E2%9C%A8-100-beta7-how-old-module-leon-package-javascript)
